### ISSUE TITLE
Error during startup `No bean named 'authenticationManager' `

### ISSUE THREAD

**Issue by egor-ryashin** [Consdata/kouncil#299]
```
2023-03-13 14:57:54 2023-03-13 11:57:54.771  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'authController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'authenticationManager' available
```

I used the latest Docker image:
```
consdata/kouncil                   latest    b0c8f4bb29ef   4 days ago      529MB
```

**Comment by ynleborg** [comment]
@egor-ryashin You have to add 
`-e kouncil.auth.active-provider="inmemory"`
while running container

**Comment by egor-ryashin** [comment]
Thank you

**Comment by pbelke** [comment]
With released version 1.6 you don't need no longer to add `-e kouncil.auth.active-provider="inmemory"`
